config/keys.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  mongoURI:
+    'mongodb+srv://appease:appease@tujue-app-cluster-tyubi.mongodb.net/test?retryWrites=true&w=majority'
```

Comment:
We need to store our database credentials in an envireonmental variable file for security purposes

Reply:
Yeah sure, let me do that, however, I had thought that you guys need to know the credentials before I have them set to the environment variable.